guacamole/src/main/frontend/src/translations/en.json

```diff
@@ -741,6 +742,7 @@
         "FIELD_HEADER_PASSPHRASE"    : "Passphrase:",
         "FIELD_HEADER_PORT"          : "Port:",
         "FIELD_HEADER_PRIVATE_KEY"   : "Private key:",
+	"FIELD_HEADER_PUBLIC_KEY"    : "Public key:",
```

Comment:
Eek! A tab!

Otherwise, LGTM.

Reply:
Whoa, not sure how that snuck in there. Fixed via rebase.